zebra-rpc/src/server/http_request_compatibility.rs

```diff
@@ -3,7 +3,8 @@
 //! These fixes are applied at the HTTP level, before the RPC request is parsed.
 
 use futures::TryStreamExt;
-use hyper::{body::Bytes, Body};
+use hyper::body::Bytes;
+use jsonrpc_http_server::{hyper, RequestMiddlewareAction};
```

Comment:
Nitpick/Optional: Could we unify the `jsonrpc_http_server` imports, import the parts needed from hyper via the re-export from `jsonrpc_http_server`, and remove `hyper` as a direct dependency in `zebra-rpc`?

```suggestion
use jsonrpc_http_server::{
    hyper::{body::Bytes, header, Body, Request},
    RequestMiddleware, RequestMiddlewareAction,
};
```

Reply:
Thanks. I was supposed to do this and forgot at the end. Done now in https://github.com/ZcashFoundation/zebra/pull/8692/commits/007b99ea411af74bd181f7d632e55f104475a989